src/mscorlib/src/System/BitConverter.cs

```diff
@@ -457,7 +457,31 @@ unsafe public static double ToDouble (byte[] value, int startIndex)
             // If we ever run on big endian machines, produce two versions where endianness is specified.
             Contract.Assert(IsLittleEndian, "This method is implemented assuming little endian with an ambiguous spec.");
             return *((double*)&value);
-        }                    
+        }
+
+        [SecuritySafeCritical]
+        public static unsafe int SingleToInt32Bits(float value)
+        {
+            // If we're on a big endian machine, what should this method do?  You could argue for
+            // either big endian or little endian, depending on whether you are writing to a file that
+            // should be used by other programs on that processor, or for compatibility across multiple
+            // formats.  Because this is ambiguous, we're excluding this from the Portable Library & Win8 Profile.
+            // If we ever run on big endian machines, produce two versions where endianness is specified.
+            Contract.Assert(IsLittleEndian, "This method is implemented assuming little endian with an ambiguous spec.");
+            return *((int*)&value);
+        }
+
+        [SecuritySafeCritical]
+        public static unsafe float Int32BitsToSingle(int value)
+        {
+            // If we're on a big endian machine, what should this method do?  You could argue for
```

Comment:
This Assert/comment is bogus. It would be better to not add it here; and delete it from the two other methods instead.